.coveragerc

```diff
@@ -7,3 +7,4 @@ skip_covered = True
 exclude_lines =
     if __name__ == .__main__.:
     if self._debug:
+    pragma: no cover
```

Comment:
I have no issues with using  `no cover`, we've just tended to use trivial pure python assertions to give the test functions coverage (also means we have some validation for the test functions). Either way seems reasonable.

Reply:
That's fine, but to test the functions that do the actual jitting (just like in this module), we need this sort of mechanism or the coverage thing is going to pester us